modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -99,17 +106,19 @@ class Numeric extends Component {
 	_handleFieldChanged(event) {
 		const value = event.target.value;
 
-		this.setState(
-			{
-				value
-			},
-			() =>
-				this.emit('fieldEdited', {
-					fieldInstance: this,
-					originalEvent: event,
+		if (value.substr(-1) != this.symbols.decimalSymbol) {
```

Comment:
@alinedoleron We can now do early return instead of wrapping the entire method inside an `if` statement